dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResourceHelper.java

```diff
@@ -141,7 +145,7 @@ public PageView getPageMetadataRendered(final HttpServletRequest request, final
         return getPageMetadata(request, response, user, uri, true, PageMode.get(request));
     }
 
-    
+    @CloseDB
     public String getPageRendered(final HttpServletRequest request, final
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1142)